sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -422,9 +436,16 @@ internal static IList<PiiTask> ConvertFromPiiEntityOptionsToTasks(IReadOnlyColle
             return list;
         }
 
-        internal static AnalyzeBatchActionsResult ConvertToAnalyzeOperationResult(AnalyzeJobState jobState, IDictionary<string, int> map)
+        internal static IList<SentimentAnalysisTask> ConvertFromAnalyzeSentimentOptionsToTasks(IReadOnlyCollection<AnalyzeSentimentOptions> analyzeSentimentOptions)
         {
-            return new AnalyzeBatchActionsResult(jobState, map);
+            List<SentimentAnalysisTask> list = new List<SentimentAnalysisTask>();
+
+            foreach (AnalyzeSentimentOptions option in analyzeSentimentOptions)
+            {
+                list.Add(ConvertToSentimentAnalysisTask(option));
+            }
+
+            return list;
```

Comment:
I think this could be replaced by calling map with `ConvertToSentimentAnalysisTask`.

Reply:
could you give me an example? also, will that be more performant than what we already have?

Reply:
I do not know the exact syntax but it should something along the lines of `analyzeSentimentOptions.map(ConvertToSentimentAnalysisTask)`.

Reply:
IReadOnlyCollection (which is the type of `analyzeSentimentOptions` doesn't have `map` methods.
I am going to leave as is unless u feel very strongly about this